pom.xml

```diff
@@ -1083,25 +1093,25 @@
   <dependency>
     <groupId>com.fasterxml.jackson.core</groupId>
     <artifactId>jackson-core</artifactId>
-    <version>2.10.1</version>
+    <version>2.12.6</version>
```

Comment:
I don't see this updated in the PollyMeta or the WikiDeataFetcherMeta files..  please generate the pom.

Reply:
the metas were updated and regenned in a later pr